tests/unit/mock/config/compliance/compliance/citrix_netscaler/netscaler_basic_feature.py

```diff
@@ -0,0 +1,3 @@
+features = [
+    {"name": "user", "ordered": False, "section": ["set system user "]},
```

Comment:
Can you add a test for `add system cmdPolicy`? This has regex in the line and would be good to make sure nothing gets tripped up.

Reply:
Also one with an add and bind, such as `add ssl` and `bind ssl`

Reply:
Done